### PR TITLE
Use thumbnail param for StaticAppBar, TopAppBar for avatar#2284

### LINKED ISSUE #2284: Use thumbnail end point for StaticAppBar, TopAppBar for avatar
**Actual Behaviour**

Not present. Uploading avatar results in a pixelated image in the account section, fix that

**Expected Behaviour**

Use thumbnail endpoint for StaticAppBar, TopAppBar for the avatar, once https://github.com/fossasia/susi_server/pull/1276 is merged. 

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2284

Changes: Used different avatar size for topBar, Accounts settings.

Demo Link: https://pr-2291-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd The frontend part is done, but the deployed server is not showing the reflected changes. I tested the changes of API its working fine locally. Changes in PR: https://github.com/fossasia/susi_server/pull/1276

File: src/redux/reducers/app.js [hunk 1]
```diff
@@ -100,6 +102,7 @@ export default handleActions(
       return {
         ...state,
         avatarImg: getUserAvatarLink(),
+        avatarImgThumbnail: getUserAvatarLink(true),
```
Comment (akshatnitd) [inline src/redux/reducers/app.js]: Make it `getUserAvatarLink({thumbnail=true})`

File: src/utils/getAvatarProps.js [hunk 2]
```diff
@@ -2,11 +2,12 @@ import md5 from 'md5';
 import { urls } from './';
 import store from '../store';
 
-export const getUserAvatarLink = () => {
+export const getUserAvatarLink = (getThumbnail = false) => {
```
Comment (akshatnitd) [inline src/utils/getAvatarProps.js]: Make changes accordingly here too.

**Comment by fragm3** [comment]
@akshatnitd made the changes.

**Pull request merged**